### ISSUE TITLE
AnalyzeAPK: xml.parsers.expat.ExpatError: duplicate attribute   (ordinal not in range(128))

### ISSUE THREAD

**Issue by wdormann** [androguard/androguard#295]
Some APKs fail to be parsed with AnalyzeAPK

### Describe what you wanted to do

I've seen this error with 9 APKs.   An example:

a,d,dx = AnalyzeAPK('com.planetcoops.android.presetupdater.apk');

https://drive.google.com/open?id=0BzrbzLF_CbEYN0pOWUlCUl93LXc

### Describe what you expected

Parse OK, as it's currently in the Google Play store.

### Describe what actually happened

Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/androguard-3.1.0.dev0-py3.5.egg/androguard/core/bytecodes/apk.py", line 158, in _apk_analysis
    self.xml[i] = minidom.parseString(raw_xml)
  File "/usr/lib/python3.5/xml/dom/minidom.py", line 1968, in parseString
    return expatbuilder.parseString(string)
  File "/usr/lib/python3.5/xml/dom/expatbuilder.py", line 925, in parseString
    return builder.parseString(string)
  File "/usr/lib/python3.5/xml/dom/expatbuilder.py", line 223, in parseString
    parser.Parse(string, True)
xml.parsers.expat.ExpatError: duplicate attribute: line 62, column 0

### System Information

- Androguard Version: master
- Python Version: 3.5.2
- Operating System: Ubuntu 16.04

**Comment by wdormann** [comment]
Additional output after the above:

```
/home/wd/in/git/androguard/androguard/core/bytecodes/dvm.pyc in _load(self, buff)
   7609             androconf.warning("no map list ...")
   7610         else:
-> 7611             self.map_list = MapList(self.CM, self.__header.map_off, self)
   7612
   7613             self.classes = self.map_list.get_item_type("TYPE_CLASS_DEF_ITEM")

/home/wd/in/git/androguard/androguard/core/bytecodes/dvm.pyc in __init__(self, cm, off, buff)
   7491             androconf.debug("Reloading %s" % TYPE_MAP_ITEM[i.get_type()])
   7492             started_at = time.time()
-> 7493             i.reload()
   7494             diff = time.time() - started_at
   7495             minutes, seconds = float(diff // 60), float(diff % 60)

/home/wd/in/git/androguard/androguard/core/bytecodes/dvm.pyc in reload(self)
   7107                     i.reload()
   7108             else:
-> 7109                 self.item.reload()
   7110
   7111     def show(self):

/home/wd/in/git/androguard/androguard/core/bytecodes/dvm.pyc in reload(self)
   2562     def reload(self):
   2563         for i in self.methods:
-> 2564             i.reload()
   2565
   2566     def show(self):

/home/wd/in/git/androguard/androguard/core/bytecodes/dvm.pyc in reload(self)
   2425     def reload(self):
   2426         self.class_idx_value = self.CM.get_type(self.class_idx)
-> 2427         self.proto_idx_value = self.CM.get_proto(self.proto_idx)
   2428         self.name_idx_value = self.CM.get_string(self.name_idx)
   2429

/home/wd/in/git/androguard/androguard/core/bytecodes/dvm.pyc in get_proto(self, idx)
   7325             self.__cached_proto[idx] = proto
   7326
-> 7327         return [proto.get_parameters_off_value(),
   7328                 proto.get_return_type_idx_value()]
   7329

/home/wd/in/git/androguard/androguard/core/bytecodes/dvm.pyc in get_parameters_off_value(self)
   2145         if self.parameters_off_value is None:
   2146             params = self.CM.get_type_list(self.parameters_off)
-> 2147             self.parameters_off_value = '({})'.format(' '.join(params))
   2148         return self.parameters_off_value
   2149

UnicodeEncodeError: 'ascii' codec can't encode character u'\u02cb' in position 104: ordinal not in range(128)

In [2]:
```

**Comment by reox** [comment]
this error results in some systems attributes are not available in the resources list.

We need to regenerate them for every new API version (see also #252 )

**Comment by reox** [comment]
For the first error, i believe it should be fixed in #296 .

**Comment by reox** [comment]
The second issue is a duplicate of #268
I'll close this one
